client/blocks/comments/post-comment-list.jsx

```diff
@@ -224,6 +227,25 @@ class PostCommentList extends React.Component {
 		);
 	};
 
+	renderCommentManageLink = () => {
+		const { siteId, postId } = this.props;
+
+		if ( ! siteId || ! postId ) {
+			return null;
+		}
+
+		return (
+			<Button
+				className="comments__manage-comments-button"
+				href={ `/comments/all/${ siteId }/${ postId }` }
+				borderless
+			>
+				<Gridicon icon="chat" />
+				<span>{ translate( 'Manage comments', { context: 'links to Comments' } ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Manage comments' )` *ES Score: 12*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/43093)


:information_source: This string already exists without a **context**. Only add a context if the meaning of the string is very specific.